backend/cmm_helpers.ml

```diff
@@ -723,7 +723,7 @@ let rec unbox_vec128 dbg =
       match Cmmgen_state.structured_constant_of_sym s.sym_name with
       | Some (Uconst_vec128 { low; high }) ->
         Cconst_vec128 ({ low; high }, dbg) (* or keep _dbg? *)
-      | _ -> Cop (Cload (Onetwentyeight, Immutable), [cmm], dbg))
+      | _ -> Cop (Cload (Onetwentyeight_unaligned, Immutable), [cmm], dbg))
```

Comment:
Maybe add a comment at the top of this function as to why alignment cannot be assumed?